services/core/shared/core/compat/sdk_v5/blocks.js

```diff
@@ -386,6 +387,9 @@ const init = async () => {
 		// Index genesis block first
 		await getBlocks({ height: genesisHeight });
 
+		// Index genesis accounts
+		await indexGenesisAccounts(genesisHeight);
```

Comment:
Let us merge the functionality into one method perhaps: `indexGenesisBlock`. This method should then fetch the genesis block and then invoke the `indexAccountsbyAddress` and ìndexTransactions`methods directly.

```suggestion
		await indexGenesisBlock(genesisHeight);
```